app/src/main/java/woowacourse/shopping/data/goods/repository/GoodsRemoteDataSourceImpl.kt

```
@@ -0,0 +1,180 @@
+                            } catch (e: Exception) {
+                                onComplete(emptyList())
+                            }
+                        }
+                    }
+                }
+
+                override fun onFailure(
+                    call: Call,
+                    e: IOException,
+                ) {
+                }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)